Firestore/core/src/model/mutable_document.cc

```diff
@@ -14,36 +14,32 @@
  * limitations under the License.
  */
 
-#include "Firestore/core/src/model/mutable_document.h"
-
 #include <sstream>
+#include "Firestore/core/src/model/mutable_document.h"
```

Comment:
Nits:
* the `mutable_document.h` include should be first (it's the related header), i.e. undo the change;
* there has to be a blank line in-between the last standard header and the first Google header (also a blank line in-between the related header and the first standard header).

See go/cstyle#Names_and_Order_of_Includes.